src/pages/Home/components/CustomCursor/CustomCursor.module.css

```
@@ -1,11 +1,13 @@
 .cursor {
   pointer-events: none;
-  position: absolute;
+  position: fixed;
   top: 0;
   left: 0;
   z-index: 10;
   padding: 0 16px;
   color: var(--white);
+  will-change: transform;
+  transform: translate(0, 0);
```

PR 본문에서는 3D transform를 사용했다고 나와있는데 현재는 단순 2D로 사용된 것 같아요! 요걸 의도하신 거 맞으신거겠죵?

혹시 몰라 [공식문서](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate3d) 첨부합니다~! 
```suggestion
  transform: translate3d(0, 0, 0);
```

---

앗 반영이 안됐네요! 짚어주셔서 감사합니다 세라!